cli/src/main/java/com/devonfw/tools/ide/commandlet/VersionSetCommandlet.java

```diff
@@ -27,9 +27,8 @@ public class VersionSetCommandlet extends Commandlet {
   public VersionSetCommandlet(IdeContext context) {
 
     super(context);
-    addKeyword("set");
+    addKeyword("set-version");
```

Comment:
correct. This is what was planned. For further simplification just use `getName()` instead of repeating as redundant String literal. Still in draft mode so I guess you are aware that there are other commandlets to change in the same way...